### PR TITLE
Fix bug where tests where not being filtered by state correctly.

### PR DESCRIPTION
**Pull request by i386**
# Description

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/BlueTestResultContainerImplTest.java [hunk 1]
```diff
@@ -30,6 +30,26 @@
     private BlueTestResultContainerImpl container;
 
     @Test
+    public void testsFilteredByState() throws Exception {
```
Comment (michaelneale) [inline blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/BlueTestResultContainerImplTest.java]: niiiice

**Comment by michaelneale** [comment]
once build and ATH happy - :bee: from me. Always nice to see test coverage improve.

File: blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/BlueTestResultContainerImplTest.java (hunk 1 above)
Comment (michaelneale) [inline blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/BlueTestResultContainerImplTest.java]: niiiice

**Comment by i386** [comment]
Passed ATH https://ci.blueocean.io/blue/organizations/jenkins/ATH-Jenkinsfile/detail/master/2319/pipeline

**Pull request merged**